static/src/stylesheets/module/email/_vars.scss

```diff
@@ -0,0 +1,4 @@
+$form-elements-height: $gs-baseline * 3;
+$form-primary-colour: #ffffff;
+$form-secondary-colour: #aad8f1;
```

Comment:
what colour is this? is it from pasteup or a whole new one? we ought to stick to pasteup really...